### ISSUE TITLE
Need "how to get started" basic documentation in the README

### ISSUE THREAD

**Issue by AKurilin** [frontrowed/sendgrid#1]
Would be great to get new people ramped up with this library, give them usage examples.

**Comment by pseudonom** [comment]
Yes, on the todo list. In the meantime, `test/Spec.hs` provides some indication of how to use this library.

**Comment by badi** [comment]
Apologies for the spam. I was rebasing to fix typos.
